docs/proposals/controller/internal-api.proto

```diff
@@ -0,0 +1,65 @@
+syntax = "proto3";
+
+package controller;
+import "google/protobuf/empty.proto";
+
+// Represents the liveness status of an Instance
+enum Status {
+  RUNNING = 0;
+  STARTING = 1;
+  STOPPED = 2;
+  STOPPING = 3;
+  DESTROYING = 4;
+  TERMINATED = 5;
+  FAILED = 6;
+  SCHEDULING = 7;
+  SCHEDULED = 8;
+}
+
+// Represents the type of workload running
+enum Type { CONTAINER = 0; }
+
+// Represents a Node status message
+message NodeStatus {
+  string id = 1;
+  Status status = 2;
```

Comment:
You're still using the same status enum for both a Node and an Instance.
What would a `Scheduling`status for a node mean?

As I asked in my previous review (https://github.com/dev-sys-do/kudo/pull/3#discussion_r928247480):

> You should have a separate enum for the Node status.

Reply:
We divided Node and Instance `State` enum in commit 344ac7f